apps/web/components/themes/toggle-theme-nav-button.tsx

```diff
@@ -0,0 +1,23 @@
+import { SidebarMenuButton } from "@repo/ui/components/sidebar";
+import { SunMoon } from "lucide-react";
+import { useTheme } from "next-themes";
+
+const ToggleThemeNavButton = () => {
+  const { setTheme } = useTheme();
+
+  const handleToggleThemeNav = () => {
+    setTheme((prev) => (prev === "light" ? "dark" : "light"));
+  };
+
+  return (
+    <SidebarMenuButton
```

Comment:
Logically, it's more [Switch](https://ui.shadcn.com/docs/components/switch) than a button